remote_control.py

```diff
@@ -110,3 +117,13 @@ def constrain(self, val, min_val, max_val):
     def stop(self):
         logging.info("RC challenge stopping")
         self.killed = True
+
+    def dither(self, counter, speed):

```

Comment:
We'd probably want to use this method when driving motors elsewhere in other challenges. How would we achieve that?

Reply:
I was thinking that we'd split RC into a motor driving module (with mixing, dither motor output functions in it) and a joystick input module. with dither being within the motor driving part and transparent to anything interacting with it. module may not be the right word here. I guess dither could be put into its own file/library to be used independently but I don't see a need for that.

Reply:
That sounds like a reasonable approach 👍 

(and module was exactly the word I was looking for ;-) )